src/SYNFPAT.m

```diff
@@ -148,6 +148,8 @@
  d setIndex^SYNFUTL(ien,"DFN",zdfn)
  d setIndex^SYNFUTL(ien,"ICN",icn)
  ;
+ I $G(^AUPNPAT(DFN,0))="" D FIXIHS()
```

Comment:
I think the changes in this flie all need to go away. I don't see a problem in creating an entry in PATIENT/IHS in my database, and if there is, the dataloader needs to be modified, not put a bandaid here.

Reply:
Put a comment saying that this is for RPMS, not VistA.